docs/customizations/playground-examples.md

```
@@ -0,0 +1,61 @@
+title: Playground Examples API
+---
+
+<script setup>
+import { useOpenapi } from 'vitepress-openapi/client'
+
+useOpenapi({
+  spec: '/openapi-playground-examples.json',
+})
+</script>
+
+# Playground Examples
```

markdownlint-cli2 (0.17.2)

Multiple top-level headings in the same document
null

(MD025, single-title, single-h1)